src/Framework/Assert.php

```diff
@@ -2830,7 +2830,7 @@ public static function assertEqualXMLStructure(DOMElement $expectedElement, DOME
                 )
             );
 
-            for ($i = 0; $i < $expectedElement->attributes->length; $i++) {
+            for ($i = 0; $i < $expectedElement->attributes->length; ++$i) {
```

Comment:
When preferred, I could also configure to consistently use the post-increment style.

Reply:
I think I would prefer that, yes.